src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -931,7 +931,9 @@ def get_file_size_from_cache_if_exists(file_path):
 
 def get_cache_file_path(file_path):
   """Return cache file path given a local file path."""
-  if not environment.get_value('NFS_ROOT'):
+  # TODO(ochang): Completely remove NFS support.
+  if (not environment.get_value('NFS_ROOT') or
+      environment.get_value('DISABLE_NFS')):
```

Comment:
Do we need the outermost parentheses?

Reply:
it's for line continuation (see https://google.github.io/styleguide/pyguide.html#32-line-length)

Reply:
Thanks!